### DISCUSSION TITLE
What does +zero do?

### DISCUSSION THREAD

**Discussion by Strykar** [tonsky/FiraCode#1389] [Q&A] [answered]
I am trying to use the `0` (in Kitty terminal) with a `.` in it not the `/`. Setting `+zero` or `-zero` does not do it, and looking at https://github.com/tonsky/FiraCode/wiki/How-to-enable-stylistic-sets `+cv15` does not either.
What am I missing here? What does `+zero` do exactly?
I wish all these alternate styles were documented better.

**Comment by wopian** [comment]
cv15 is for * positions/style, not 0

zero is 0 with a dot
cv11 is 0 with nothing inside
cv12 is 0 with a \\ inside
cv13 is 0 with a | inside

**Comment by Strykar** [comment]
My kitty config has `font_features FiraCode-Regular +ss01 +ss02 +ss03 +ss04 +ss05 +ss07 +ss08 +zero +onum +calt` but the 0 is displayed as:
![zero](https://user-images.githubusercontent.com/2946372/158363518-546128d4-3473-49ad-9ac0-b1ecd428f180.png)
Shouldn't +zero set that to a zero with a dot?

**Reply by wopian** [reply]
Do you have an older version of Fira Code installed on your system from before stylistic sets were added in v2+?

**Reply by tonsky** [reply]
This is not Fira Code at all. Check your installation, maybe font name

**Reply by alexeyten** [reply]
Looks like Noto Sans Mono

**Reply by Strykar** [reply]
I am an idiot and had `font_family FiraCode Regular` instead of `Fira Code Regular`.
Looks good now, sorry for the wild goose chase.
![defo_fira](https://user-images.githubusercontent.com/2946372/158391694-78324b0f-7da8-428c-977f-6a2691a9255f.png)

I have one Q, what is the logic behind `onum`, is it just to fit more numbers in the same space?
I have a large 2K monitor and I realized I prefer the standard perfectly aligned way.

**Reply by tonsky** [reply]
onum enables “old-style numerals”. I believe the idea was that they start to behave like lower-case letters: some descend beyond baseline. Some might argue they look better in a text context. I personally never liked them all that much

**Comment by Strykar** [comment]
@tonsky Would you please reply in a new box so I can mark it as the answer. Thank you!

**Comment by tonsky** [comment, marked as answer]
This is not Fira Code at all. Check your installation, maybe font name
